src/surge-xt/gui/SurgeGUIEditorHtmlGenerators.cpp

```diff
@@ -411,6 +412,223 @@ th {
+{
+    std::ostringstream htmls;
+
+    htmls <<
+        R"HTML(
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Lato" />
+    <style>
+table {
+  border-collapse: collapse;
+  width: 100%;
+
+}
+
+td {
+  border: 1px solid #CDCED4;
+  padding: 2pt 4px;
+}
+
+.center {
+  text-align: center;
+}
+
+div.heading {
+    margin: 4px 0px 0px 8px;
+    width: 100%;
+}
+
+h3 {
+    margin: 0 0 8px 0;
+}
+
+div.outer {
+    font-size: 12pt;
+    margin: 0 0 10pt 0;
+    font-family: Lato;
+    color: #123463;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+div.tablewrap {
+    width: 600px;
+    margin: 0 8px 16px 8px;
+    flex-basis: 40%;
+}
+
+th {
+  padding: 4pt;
+  color: #123463;
+  background: #CDCED4;
+  border: 1px solid #123463;
+}
+
+span {
+    margin-left: 16px;
+}
+</style>
+  </head>
+  <body style="margin: 0pt; background: #CDCED4;">
+    <div style="border-bottom: 1px solid #123463; background: #ff9000; padding: 2pt;">
+      <div style="font-size: 20pt; font-family: Lato; padding: 2pt; color:#123463;">
+        Surge XT parameters addressable by OSC
+      </div>
+    </div>
+
+    <div style="margin:10pt; padding: 5pt; border: 1px solid #123463; background: #fafbff;">
+      <div style="font-size: 12pt; font-family: Lato; color: #123463;">
+        Construct OSC messages for Surge XT using the exact (case sensitive)
+        entry listed in the 'Address' column in the tables below.</br>
+        The form of the message should be <b>/param/&ltaddress&gt value</b>,
+        where 'value' is either
+        <ul>
+            <li>a floating point value between 0.0 and 1.0</li>
+            <li>an integer value</li>
+            <li>0 or 1 (boolean)</li>
+            <li>contextual: either an in integer or a float, depending on the context</li>
+        </ul>
+        Where an address is listed as beginning with <b>'*'</b>, replace the <b>'*'</b> with either <b>'a'</b> or <b>'b'</b>,
+        depending on which scene you wish to address. E.g., 'a_drift' or 'b_drift'.
+        <p>Examples: <span><b>/param/volume 0.63</b></span>
+        <span><b>/param/polylimit 12</b></span><span><b>/param/a_mute_noise 0</b></span></p>
+      </div>
+    </div>
+
+    <div style="margin:10pt; padding: 5pt; border: 1px solid #123463; background: #fafbff; overflow:hidden">
+      <div class="outer">
+    )HTML";
+
+    std::vector<oscParamInfo *> sortvector;
+    std::string st_str;
+    int currentCtrlGrp = endCG;
+
+    for (auto *p : synth->storage.getPatch().param_ptr)
+    {
+        st_str = p->get_OSC_name();
+        if (st_str[1] == '_')
+        {
+            if (st_str[0] == 'b')
+                continue; // 'b_...' entries not added to vector
+            else if (st_str[0] == 'a')
+            {
+                st_str[0] = '*';
+            }
+        }
+        sortvector.push_back(new oscParamInfo(p, st_str, p->get_full_name(), p->ctrlgroup));
```

Comment:
This is a memory leak. You're instantiating new oscParamInfos, but you're never deleting them.

Suggest you do not use "new" at all. Instead, sortVector should simply be: std::vector<oscParamInfo> sortVector.

Then do not create "new oscParamInfo(...)", but instead just "oscParamInfo(...)".

Reply:
'std::vector sortvector;' gives 'no instance of constructor "std::__1::vector" matches the argument list'

Reply:
I've changed the comparator, but still get the same error on the 'std::vector sortvector;' declaration. I don't get how it can be called without the template arg.

Reply:
I think github mangled the comment and he meant std vector open type close

I can’t do backticks on my phone. But change the vector from having type be a foo* to a foo and drop the new

Reply:
Fixed.